projects/packages/image-cdn/tests/php/Image_CDN_Core_Test.php

```diff
@@ -450,6 +450,18 @@ public function test_photonizing_check_extensions( $image_url, $expected ) {
 		$this->assertEquals( $expected, Image_CDN_Core::cdn_url( $image_url, array( 'w' => 500 ) ) );
 	}
 
+	/**
+	 * Tests that the cdn_url method returns an empty string when the image URL is empty or invalid.
+	 *
+	 * @since $$next-version$$
+	 */
+	public function test_cdn_url_empty_invalid_url() {
+		$this->assertSame( '', Image_CDN_Core::cdn_url( '' ) );
+		$this->assertSame( '', Image_CDN_Core::cdn_url( null ) );
+		$this->assertSame( '', Image_CDN_Core::cdn_url( 123 ) ); // @phan-suppress-current-line PhanTypeMismatchArgument
```

Comment:
Looks like we need to suppress Phan here.

```suggestion
		// @phan-suppress-next-line PhanTypeMismatchArgument -- This is testing unexpected types.
		$this->assertSame( '', Image_CDN_Core::cdn_url( 123 ) );
```

Reply:
Handled before my comment. 😄 5b94d7d